tests/integration/ceph_bucket_notification_test.go

```diff
@@ -71,7 +71,7 @@ func testBucketNotifications(s *suite.Suite, helper *clients.TestClient, k8sh *u
 	t.Run("create CephBucketTopic", func(t *testing.T) {
 		err := helper.TopicClient.CreateTopic(topicName, storeName, httpEndpointService)
 		assert.Nil(t, err)
-		created := utils.Retry(12, 2*time.Second, "topic is created", func() bool {
+		created := utils.Retry(12, 30*time.Second, "topic is created", func() bool {
```

Comment:
Let's retry more times and more often, instead of just increasing the retry sleep time.
```suggestion
		created := utils.Retry(30, 5*time.Second, "topic is created", func() bool {
```